docs/settings/security-settings.asciidoc

```diff
@@ -239,3 +239,114 @@ The format is a string of `<count>[ms\|s\|m\|h\|d\|w\|M\|Y]` (e.g. '20m', '24h',
 ============
 
 |===
+
+[float]
+[[audit-logging-settings]]
+===== Audit logging settings
+
+You can enable audit logging to support compliance, accountability and security. When enabled {kib} will capture:
+
+- Who performed an action
+- What action was performed
+- When it occurred
+
+For more details and a reference of audit events see <<xpack-security-audit-logging>>.
```

Comment:
```suggestion
For more details and a reference of audit events, refer to <<xpack-security-audit-logging>>.
```

Reply:
To meet accessibility standards, we've removed instances of `see`, and replaced them with `refer to`.